modules/nf-core/gt/suffixerator/tests/nextflow.config

```diff
@@ -0,0 +1,3 @@
+process {
+    ext.args = '-dna -des -sds no'
```

Comment:
Are these options necessary for the tool to run, or are they here just because of the dataset used?

Reply:
If the module would not function without certain flags these should be supplied through `input:` to ensure a minimally functioning module. e.g. `BUSCO` always needs a `-mode` value so it comes in through `input:`.

Reply:
Thank you @mahesh-panchal 

I totally forget about this practice. I have now added `mode` as input and also a few additional tests.